opslevel/resource_opslevel_webhook_action.go

```diff
@@ -63,6 +65,10 @@ func (r *WebhookActionResource) Schema(ctx context.Context, req resource.SchemaR
 		MarkdownDescription: "WebhookAction resource",
 
 		Attributes: map[string]schema.Attribute{
+			"async": schema.BoolAttribute{
+				MarkdownDescription: "Whether or not the Webhook Action expects an additional, asynchronous response upon completion.",
+				Required:            true,
```

Comment:
So this is the most contentious part.  As is this means when customers upgrade it'll be a breaking change so we'll have to do a minor version bump.

If we want it to not be a breaking change we could modify it to be

```suggestion
				Optional:            true,
				Default:             booldefault.StaticBool(false),
```

This should preserve how it worked prior to this change and not cause customers existing configuration to blow up once they upgrade provider versions.  The trade off is - does this work properly down in Create and Update.

Reply:
This is actually more true to the create & update behaviour too